src/ts/component/page/elements/head/simple.tsx

```diff
@@ -133,29 +166,33 @@ const HeadSimple = observer(class Controls extends React.Component<Props> {
 		};
 
 		return (
-			<div ref={node => this.node = node} className={cn.join(' ')}>
-				<div className="side left">
-					<div className="titleWrap">
-						{!noIcon && check.withIcon ? (
-							<IconObject 
-								id={'block-icon-' + rootId} 
-								size={32} 
-								iconSize={32}
-								object={object} 
-								canEdit={canEditIcon} 
-							/>
-						) : ''}
-						<Editor className="title" id="title" />
+			<div className="headSimpleWrapper">
```

Comment:
What is this wrapper for?

Reply:
It is for styling and positioning week day name as designed. I'd prefer to move it inside headSimple but doing this could be harmful as headSimple class is used in a lot of places.